eXo/Sources/Controllers/Basic/eXoWebBaseViewController.swift

```diff
@@ -53,7 +53,7 @@ class eXoWebBaseController: UIViewController {
         // Add a custom values to the default user agent
         webView?.evaluateJavaScript("navigator.userAgent") { (result, error) in
             if let userAgent = result as? String {
-                self.webView?.customUserAgent = userAgent + "\(Bundle.main.appName)/\(Bundle.main.versionNumber) Version/\(UIDevice.current.systemVersion) Safari/604.1 (iOS)"
+                self.webView?.customUserAgent = "eXo/" + userAgent + "\(Bundle.main.appName)/\(Bundle.main.versionNumber) Version/\(UIDevice.current.systemVersion) Safari/604.1 (iOS)"
```

Comment:
What does this part of code print ? doesn't it show the application name and version ? i.e eXo/6.2 etc ... 
``` \(Bundle.main.appName)/\(Bundle.main.versionNumber) ```